### PR TITLE
Registering of kl-divergence for independent distribution

### PR DESCRIPTION
**Pull request by SkafteNicki**
This address issue https://github.com/pytorch/pytorch/issues/13545 and implements the proposed fix together with a single test.

### CODE REVIEW

File: torch/distributions/travis.log [hunk 1]
```diff
@@ -0,0 +1,87 @@
+Command line:
```
Comment (vishwakftw) [inline torch/distributions/travis.log]: Please remove this file.

**Comment by ezyang** [comment]
@vishwakftw I'll leave the rest of the review to you; I went ahead and deleted the file tho :>

**Review by vishwakftw — APPROVED** [review]
LGTM

cc: @fritzo

**Review by fritzo — APPROVED** [review]
Looks great. Thanks for adding this!

**Comment by ezyang** [comment]
@vishwakftw Tip: requesting a "rebase this please" will cancel an ongoing land; so no need to do so unless you're trying to get more up to date test signal ;)

**Comment by vishwakftw** [comment]
Oh, I’m sorry about that. The PR branch was extremely outdated, which is why I did that. I’ll take care next time.

**Comment by SkafteNicki** [comment]
Sorry about the inconvenience of me using a old branch. This is the first time I am creating a pull request so I am still learning about how to do it probably.

**Pull request closed without merging**